eng/pipelines/swagger-api-doc-preview.yml

```diff
@@ -0,0 +1,203 @@
+trigger: none
+pr:
+  paths:
+    include:
+      - specification/**
```

Comment:
Should we also trigger on the check yaml and source code, as a smoke test when we change something?